test/run-php-linter.sh

```diff
@@ -3,7 +3,7 @@ set -euo pipefail
 
 # Run PHP -l on everything to ensure there's no syntax
 # errors.
-for i in `ls php/libraries/*.class.inc modules/*/php/* modules/*/ajax/* htdocs/*.php htdocs/*/*.php`;
+for i in php/libraries/*.class.inc modules/*/php/* modules/*/ajax/* htdocs/*.php htdocs/*/*.php;
 do
   php -l $i >/dev/null || exit $?;
```

Comment:
With "set -euo pipefail" there is no need for || exit $?, as any error in the script will trigger an exit with error from the script.